consumer_group.go

```diff
@@ -440,11 +439,10 @@ func (cg *ConsumerGroup) assignPartitions(topic string) ([]int32, error) {
 	}
 	partitions := make([]int32, 0, partNum/consumerNum+1)
 	for i := 0; i < partNum; i++ {
```

Comment:
if the partNum is int32 instead of int, this `for` loop could be changed into:
for i := int32(0); ....